src/HttpKernel/ContaoCache.php

```diff
@@ -42,4 +47,15 @@ public function fetch(Request $request, $catch = false): Response
     {
         return parent::fetch($request, $catch);
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function createStore()
+    {
+        return new Psr6Store([
+            'cache_directory' => $this->cacheDir,
+            'cache_tags_header' => 'X-Cache-Tags',
```

Comment:
Aren't X-Headers deprecated? Or is this a requirement by the FOSCacheBundle?

Reply:
It's just the default header FOSCache uses by default (and cannot be changed for BC reasons). It can be configured but I think it's not worth the effort.